src/fs_io.h

```diff
@@ -30,16 +30,26 @@ class dread_ctrl
     void *client_data;
 };
 
-// POD
 class dwrite_q
 {
+    MEMPROXY_CLASS(dwrite_q);
 public:
-    off_t file_offset;
-    char *buf;
-    size_t len;
-    size_t buf_offset;
-    dwrite_q *next;
-    FREE *free_func;
+    dwrite_q() = default;
```

Comment:
If this constructor is currently unused, please do not add it:

```suggestion
```

Otherwise, please point me to where it is used.

Reply:
Removed in 9cb4fbbda2 fixes.